sections/semantics-root.include

```diff
@@ -38,35 +37,9 @@
 
   The <{html}> element <a>represents</a> the root of an HTML document.
 
-  Authors are encouraged to specify a <{global/lang}> attribute on the root <{html}>
-  element, giving the document's language. This aids speech synthesis tools to determine what
-  pronunciations to use, translation tools to determine what rules to use, and so
-  forth.
-
-  <p class="note">
-    It is recommended to keep the usage of attributes and their values defined on the <{html}>
-    element to a minimum to allow for proper detection of the <a>character encoding declaration</a>
-    within the first 1024 bytes.
```

Comment:
I think the above paragraph should not be removed; this isn't about app-cache, but character encoding, and is still good advice.